articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -102,13 +102,12 @@ use the **TableBatch** class. When you do want to submit the
 batch, you call **commit\_batch**. Note that all entities must be in the same partition in order to be changed as a batch. The example below adds two entities together in a batch.
 
 ```python
-from azure.storage.table import TableBatch
-batch = TableBatch()
+batch = table_service.begin_batch()
```

Comment:
@asears Hi Andrew, thanks again for contributing. I'm checking in to see if you've had a chance to review my reply above. If you wish to proceed with this PR, please either update the code snippet so that it no longer throws the `AttributeError`, or let me know which dependencies you've satisfied to allow your proposed code to function as-is.

If you no longer wish to proceed with this PR, please either close the PR or let me know and I can have it closed for you. Thanks! --Marsh